init/central-ledger/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM postman/newman:5.0.0-alpine
+LABEL maintainer="kamuela.franco@modusbox.com"
+
+WORKDIR /opt/init-central-ledger
+COPY . .
+
+RUN ["node", "inject-env-into-postman-env.js"]
+
+RUN newman run \
+    --delay-request=3000 \
+    --environment="Onboard-Generic-FSP-POPULATED.postman_environment.json" \
+    "Onboard-Generic-FSP.postman_collection.json"
```

Comment:
Yes, you can't use `CMD` more than once. But I always took it as the "server." Since this isn't a long-running process, should I really be trying to use it?

Reply:
`RUN` only runs when the image is built. If you try to run this image it will probably do nothing as you haven't actually specified a runtime command. (Maybe it'll start a shell or something, whatever the base image command was).